nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java

```diff
@@ -321,6 +328,15 @@ public void process(final InputStream in) throws IOException {
                                 attributes.put(CoreAttributes.ABSOLUTE_PATH.key(), absPathString);
                                 attributes.put(CoreAttributes.MIME_TYPE.key(), OCTET_STREAM);
 
+                                attributes.put(FILE_INNER_PERMISSION, String.valueOf(tarEntry.getMode()));
+                                attributes.put(FILE_INNER_OWNER, String.valueOf(tarEntry.getUserName()));
+                                attributes.put(FILE_INNER_GROUP, String.valueOf(tarEntry.getGroupName()));
+
+                                String timePattern = "yyyy-MM-dd'T'HH:mm:ssZ";
+                                DateFormat df = new SimpleDateFormat(timePattern);
```

Comment:
Instead of `SimpleDateFormat`, `DateTimeFormatter` should be used in Java 8+ which is thread safe and can be defined as a constant.
The built-in `DateTimeFormatter.ISO_OFFSET_DATE_TIME` instance seems to me the right format here.

Reply:
> The built-in `DateTimeFormatter.ISO_OFFSET_DATE_TIME` instance seems to me the right format here.

Some other processors (ListFTP, ListFile, PutFile, GetFile) use "yyyy-MM-dd'T'HH:mm:ssZ" format explicitly.